<launch>
  <node pkg="mocap_qualisys"
    type="mocap_qualisys_node"
    name="qualisys"
    output="screen">
    <param name="server_address" value="192.168.129.216"/>
    <param name="server_base_port" value="22222"/>
    <param name="frame_rate" value="100"/>
    <param name="max_accel" value="10.0"/>
    <param name="publish_tf" value="true"/>
    <param name="fixed_frame_id" value="mocap"/>
    <rosparam param="model_list">[]</rosparam>
    <!--remap from="qualisys/F450/odom" to="/f450/odom"/-->
  </node>
</launch>
